scss/vendor/reset.css

```diff
@@ -0,0 +1,43 @@
+html, body, div, span, applet, object, iframe,
```

Comment:
When copying code - leave a comment with a ref to a original.
Leave empty line in the end of each file